### DISCUSSION TITLE
Attempt to output variable RS (total stochastic resistance)or LAISUN and LAISHA

### DISCUSSION THREAD

**Discussion by Pengzhuohoa** [NCAR/noahmp#128] [Q&A] [answered]
Hello
I am trying to output variables RS (total stochastic resistance) or LAISUN and LAISHA in wrf chem, and I am working on modules_sf-noahmplsm Attempts were made to modify the F file, but none were successful, and the variables outputted during the modification in the Registry.EM-COMMON file of wrf did not have values.
If you could provide some guidance, I would be grateful

**Comment by cenlinhe** [comment, marked as answer]
Hi, this may be better posted on the WRF discussion forum, but anyway to answer your question, you will need to modify registry.noahmp instead of Registry.EM_COMMON. The RSSUN and RSSHA are already there in registry.noahmp, you just need to add them to the history file. For LAISUN and LAISHA, you can follow exactly RSSUN and RSSHA to modify the F files where these variables occurring within the WRF.

**Comment by Pengzhuohoa** [comment]
Thank you for your suggestion. I will try it out and thank you again for your guidance
